requirements.txt

```diff
@@ -7,14 +7,14 @@ docopt==0.6.2
 Flask==1.1.2
 Flask-Routing==0.0.21
 idna==2.10
-internetarchive==1.9.4
+internetarchive==2.1.0
```

Comment:
2.2.0 ?  https://pypi.org/project/internetarchive

Reply:
We removed the dependencies and let pip install the right matches and these are the changes it proposed, so I'm going to go with them for now